telemetry/definitions/commonDefinitions.json

```diff
@@ -1848,6 +1872,29 @@
                 }
             ]
         },
+        {
+            "name": "auth_userState",
+            "description": "The state of the user's authentication.",
```

Comment:
metrics are for events. state isn't an event

What event does auth_userState represent? It looks more like data. Can the data be stored on an existing event? Otherwise, it's not clear when this "event" should be emitted, it would be more like a heartbeat (which then implies it should be on... an existing, semi-frequent event)

Reply:
it's only on startup.

Reply:
I see that this is upstreaming an existing metric: https://github.com/aws/aws-toolkit-vscode/blob/0f9782a9d8407d3d755c44dc3a3eec1fe405e7b9/packages/core/src/shared/telemetry/vscodeTelemetry.json#L1108-L1109

Reply:
yes Ideally it should be in common only because JetBrains is also emitting this metric. Trying to fix it by moving it to common.